jsapp/js/components/librarySidebar.es6

```diff
@@ -210,6 +210,15 @@ class LibrarySidebar extends Reflux.Component {
       assetid: collectionUid
     });
   }
+  isCollectionPublic(collection) {
+    let isPublic = false;
+    collection.permissions.forEach((perm) => {
```

Comment:
Any reason not to use [`getAnonymousUserPermission()`](https://github.com/kobotoolbox/kpi/blob/7255202c0729a884a3cecb3ea4378fd1c448029c/jsapp/js/utils.es6#L35-L42)? There's an example of it being called in this file, [a few lines below](https://github.com/kobotoolbox/kpi/blob/7255202c0729a884a3cecb3ea4378fd1c448029c/jsapp/js/components/librarySidebar.es6#L225).

Reply:
The reason is me not knowing what this function does 👎 Fixed

Reply:
👎🔄👍